%YAML:1.0

BaseLidar:
  pcds_file: /mount_dir/lam_data/calibration_jira/DG_0725/pcd
  max_dis: 40
  min_dis: 1
  # hesai64, bp32, helios32, ft120, yijing
  lidar_type: ft120 
  tz: 1.2

RefNum: 0
Step: 5
SceneNum: 3
ThTime: 3

ShowResidual: 1
UseCADPrior: 1

DrPosesFile: /mount_dir/lam_data/calibration_jira/DG_0725/video/24-07-26_10-44-09_268/video/dr_tum.txt
AroundCameraStampFile: /mount_dir/lam_data/calibration_jira/DG_0725/video/24-07-26_10-44-09_268/video/video_frame_info
ImageFilesPath: /mount_dir/lam_data/calibration_jira/DG_0725/video/24-07-26_10-44-09_268/jpg
CamsIntrinsicPath: /home/CN/xiang.zhao/ws_dropngo_collection/dng_flashbot_2.2_cget/sources/lam/01_preresearch_calibration/dr_lidar_calib/#output/camera_bk/
CamsExtrinsicPath: /home/CN/xiang.zhao/ws_dropngo_collection/dng_flashbot_2.2_cget/sources/lam/01_preresearch_calibration/dr_lidar_calib/#output/camera_bk/
LidarExtrinsicFile: /home/CN/xiang.zhao/ws_dropngo_collection/dng_flashbot_2.2_cget/sources/lam/01_preresearch_calibration/assets/CarModels/CN01/lidar/roof_front_transform.pb.txt

SaveLidarExtrinsicName: roof_front_transform.pb.txt
ResultPath:  /home/CN/xiang.zhao/ws_dropngo_collection/dng_flashbot_2.2_cget/sources/lam/01_preresearch_calibration/dr_lidar_calib/#output


#####
CameraName:
  - avm_front
  - avm_left
  - avm_rear
  - avm_right
  # - front_near
  # - left_front
  # - left_rear
  # - rear_medium
  # - right_front
  # - right_rear

Canny.gray_threshold: 10
Canny.len_threshold: 50

# Params for Voxel Cutting & Plane Fitting & Edge Extraction
UseAdaVoxel: 0
Voxel.size: 0.6
# Voxel.size: 0.5
Voxel.eigen_ratio: 0.05
Plane.min_points_size: 30
Plane.normal_theta_min: 20
Plane.normal_theta_max: 170
Ransac.dis_threshold: 0.02
# Ransac.dis_threshold: 0.015
Edge.min_dis_threshold: 0.08
# Edge.min_dis_threshold: 0.05
